README.md

```diff
@@ -1,6 +1,29 @@
 # Music Library
 
-This project is part of the Backend Module of the Manchester Codes Bootcamp Syllabus. 
+This project is part of the Backend Module of the Manchester Codes Bootcamp Syllabus.
```

Comment:
This is a pretty good readme, but there are a few things missing:
- A description of what the app actually does and what its features are,
- installation instructions (especially regarding the database and the need to create environment variables). The shell script is a nice touch for starting the docker container, but some details about this should be included in here,
- no mention of the need for `.env` and `.env.test` files (and what values they should provide),
- information about the routes provided by the app, what type of HTTP verb they all require and an example request body would be a great addition,
- how to test the app